R/lockbox.R

```diff
@@ -133,7 +133,7 @@ lockbox_download_dir <- function() {
 #' The transient lockbox library path.
 lockbox_transient_dir <- function() {
   getOption("lockbox.transient_dir") %||%
-    normalizePath("~/.R/lockbox_transient", mustWork = FALSE)
+    normalizePath(paste0("~/.R/", .get_safely_from_envir("session_id")), mustWork = FALSE)
```

Comment:
I was having trouble reading the intent on `lockbox_transient`.  Does the useR expect the `lockbox.transient_dir` to point unadulterated to their transient directory or do they think it is the root where we will dump sessions?  I was thinking the former.